tests/unit/neptune/new/test_handler.py

```diff
@@ -270,15 +271,15 @@ def test_log_many_values(self):
 
     def test_append_many_values_cause_error(self):
```

Comment:
It sould not be called `test_append_many_values_cause_error`, now error is caused by non-supported value.
We could test some `set`, `tuple` and instance of custom class here as well.

Reply:
Done